backend/hostgw/hostgw_windows.go

```diff
@@ -160,26 +160,6 @@ func (be *HostgwBackend) RegisterNetwork(ctx context.Context, wg sync.WaitGroup,
 			return nil, errors.Annotatef(err, "failed to create HNSNetwork %s", networkName)
 		}
 
-		// Wait for the network to populate Management IP
-		log.Infof("Waiting to get ManagementIP from HNSNetwork %s", networkName)
-		waitErr = wait.Poll(500*time.Millisecond, 5*time.Second, func() (done bool, err error) {
-			newNetwork, lastErr = hcsshim.HNSNetworkRequest("GET", newNetwork.Id, "")
-			return newNetwork != nil && len(newNetwork.ManagementIP) != 0, nil
```

Comment:
Can you add a nil check for newNetwork.ManagementIP here and see if that resolves the error? This may be a better solution

Reply:
Also try increasing the timeout to 2 minutes with this change

Reply:
@song-jiang There is a known issue in RS4 that the ManagementIP may not be populated until the host receives an IP. This does not occur in RS5 onwards. Can you try delaying starting flannel until the host receives an IP? I think flannel may be starting too soon

Reply:
@ksubrmnn I can confirm the host did receive an IP before starting flannel process. I used powershell script similar to https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/l2bridge/start.ps1 with small modifications. 

The flanneld process is started after
- `flanneld.exe` binary downloaded from internet.
- Docker images pulled.
- windows node registered to kube-apiserver. 
- vSwitch created.

Reply:
Is it possible to get access to your set up? This issue should not happen and we have not been able to repro it.

Reply:
@ksubrmnn I will try to reproduce and come back to you. Could you email your contact information to song@tigera.io for me to send you access credentials? Thanks.